### PR TITLE
add profiles.photos field

### PR DESCRIPTION
**Pull request by antonlukin**
Suggest to add photos field to profile object for consistency.
Related docs: https://yandex.ru/dev/passport/doc/dg/reference/response-docpage/

### CODE REVIEW

File: lib/passport-yandex/strategy.js [hunk 1]
```diff
@@ -102,6 +102,12 @@ Strategy.prototype.userProfile = function(accessToken, done) {
       profile.gender = json.sex;
       profile.emails = [{ value: json.default_email }];
 
+      if (json.default_avatar_id) {
+        profile.photos = [{
+          value: 'https://avatars.yandex.net/get-yapic/' + json.default_avatar_id + '/islands-200'
+        }];
+      }
```
Comment (gurugray) [inline lib/passport-yandex/strategy.js:109]: please, add `thumbnail` type

```js
      if (json.default_avatar_id) {
        profile.photos = [{
          value: 'https://avatars.yandex.net/get-yapic/' + json.default_avatar_id + '/islands-200',
          type: 'thumbnail'
        }];
      }
```
according http://portablecontacts.net/draft-spec/

**Comment by gurugray** [comment]
add please new field in typings
https://github.com/gurugray/passport-yandex/blob/master/typings/index.d.ts#L9

**Comment by gurugray** [comment]
and there are test needed for this field https://github.com/gurugray/passport-yandex/blob/master/test/strategy-test.js

**Comment by antonlukin** [comment]
Thank you for such a quick response. I tried to fix everything you pointed out. Hope I got that right.

**Comment by gurugray** [comment]
merged, thanks for your contribute!

**Pull request closed without merging**